### DISCUSSION TITLE
v0.2.0 - Harbor App

### DISCUSSION THREAD

**Discussion by av** [av/harbor#42] [Announcements]
### v0.2.0 - Harbor App

![Harbor App UI screenshot](https://github.com/av/harbor/wiki/harbor-app-3.png)

Harbor App is a companion application for [Harbor](https://github.com/av/harbor) - a toolkit for running AI locally. It's built on top of functionality provided by the `harbor` CLI and is intended to complement it.

The app provides clean and tasteful UI to aid with workflows associated with running a local AI stack.

- 🚀 Manage Harbor stack and individual services
- 🔗 Quickly access UI/API of a running service
- ⚙️ Manage and switch between different configurations
- 🎨 Aestetically pleasing UI with plenty of built-in themes

## Demo

https://github.com/user-attachments/assets/a5cd2ef1-3208-400a-8866-7abd85808503

In the demo, Harbor App is used to launch a default stack with [Ollama](./2.2.1-Backend:-Ollama) and [Open WebUI](./2.1.1-Frontend:-Open-WebUI) services. Later, [SearXNG](./2.3.1-Satellite:-SearXNG) is also started, and WebUI can connect to it for the Web RAG right out of the box. After that, [Harbor Boost](./5.2.-Harbor-Boost) is also started and connected to the WebUI automatically to induce more creative outputs. As a final step, Harbor config is adjusted in the App for the [`klmbr`](./5.2.-Harbor-Boost#klmbr---boost-llm-creativity) module in the [Harbor Boost](./5.2.-Harbor-Boost), which makes the output unparseable for the LLM (yet still undetstandable for humans).

## Installation

Please refer to the [Installation Guide](https://github.com/av/harbor/blob/v0.2.0/1.0.-Installing-Harbor#harbor-app)

<hr /><em>This discussion was created from the release <a href='https://github.com/av/harbor/releases/tag/v0.2.0'>v0.2.0 - Harbor App</a>.</em>

**Comment by av** [comment]
This is a very first release of the Harbor App, things might not work as expected - don't hesitate to post

**Comment by SimonBlancoE** [comment]
Can you clarify if the app can be accessed remotely via a web interface? If not, is there any plan to implement it?

Right now I can remote desktop into my server, but in case I couldn't, I'd like to know if it's possible.
